### ISSUE TITLE
Questions About Autoencoder Training: Loss Function, Training Steps, and Learning Rate Scheduler

### ISSUE THREAD

**Issue by SerdarHelli** [Seed3D/Dora#3]
Which loss function did you use when training the autoencoder? Did you directly compute the MSE between the predicted SDF values and the ground truth? Additionally, how many steps did you train for? Did you use a learning rate scheduler? If so, which one did you use?

These are questions I've always been curious about, as they're often not fully detailed in the paper.

**Comment by SerdarHelli** [comment]
I could replicate the model, it worked well.

**Comment by melo513** [comment]
> I could replicate the model, it worked well.

can you share your training parameters?
